plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareManager.java

```diff
@@ -46,6 +46,10 @@
     static final ConfigKey<Integer> s_vmwareOVAPackageTimeout = new ConfigKey<Integer>(Integer.class, "vmware.package.ova.timeout", "Advanced", "3600",
             "Vmware script timeout for ova packaging process", true, ConfigKey.Scope.Global, 1000);
 
+    public static final ConfigKey<Integer> VMWARE_STATS_TIME_WINDOW = new ConfigKey<Integer>("Advanced", Integer.class, "vmware.stats.time.window", "300",
```

Comment:
```suggestion
    public static final ConfigKey<Integer> s_vmwareStatsTimeWindow = new ConfigKey<Integer>("Advanced", Integer.class, "vmware.stats.time.window", "300",
```

better to keep same format as other configs

Reply:
Hi @sureshanaparti, thanks for the review. About changing this to keep the format at the other configs, I really don't think this is a good action. This config is a constant, and, according to [Java naming conventions](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html), and [ACS naming conventions](https://cwiki.apache.org/confluence/display/CLOUDSTACK/Coding+conventions) , constants must be written in uppercase, separated by _. Keeping new settings in the old format does not sound like a good practice.